### ISSUE TITLE
Right pane shows raw git diff output only instead of history code with coloured highlights

### ISSUE THREAD

**Issue by pamtbaau** [jakesankey/git-history#30]
When picking a commit from history, a second panel is opened and filled with the raw output of git.

What I expected was two panes with current and history code and coloured highlights of the differences, kinda like [split-diffs](https://atom.io/packages/split-diff) does.

Am I missing something?

**Comment by jakesankey** [comment]
Did you check the package settings? There is an option to show either history or diff

**Comment by pamtbaau** [comment]
Ah, I took that option to mean that a second panel should be opened to show the differences...

After deselecting the Show Diff option I do get two panels with the 'before/after', but don't see differences highlighted. That was my expectation...

**Comment by jakesankey** [comment]
Do you have language-diff package installed and enabled? That is how atoms knows how to highlight the file. 

Jake

**Comment by pamtbaau** [comment]
Just installed language-diff, but doesn't make a difference... Two panes from different commits sittings next to each other, but no highlights of differences and no sync scrolling.

Do I have the wrong expectations?

**Comment by jakesankey** [comment]
Sync scrolling is not (yet) a feature of this plugin but diff highlighting is. No magic there. The right pane should be a .diff file and atom should highlight that automatically. Try restarting atom after installing language-diff. Give it one more shot. If that doesn't work. I'll look into it more.

**Comment by pamtbaau** [comment]
Nope, no highlights. I used default settings for `language-diff`. 
The right-hand side file by the way is not called `.diff` which `language-diff` is said to work on. The file has the same filename as on the left side, prepended with a number.

By the way... is it an idea to clear the folder `C:\Users\pamtbaau\.atom\.git-history` with all its files on deactivation of your package?

**Comment by jakesankey** [comment]
Interesting. If you have the "show diff" option on in git-history, it should be a .diff file on the right. I can look into this further. Thanks for the info. 

It wouldn't hurt to wipe that .git-history file and see what happens if you wanna try that. 

Jake

**Comment by pamtbaau** [comment]
If setting Show Diff is toggled on
     the right-hand side has the .diff extension, but the pane shows the raw diff.
else
     the right-hand side has the same original extension and panel shows code

**Comment by jakesankey** [comment]
By "raw diff" you mean it doesn't include highlighting? It is supposed to show the git diff but it should highlight.

**Comment by pamtbaau** [comment]
Maybe its a good idea to add some images...

Show Diff = off (no highlighting of differences)
![show diff - off](https://cloud.githubusercontent.com/assets/5120011/11429910/9cd24a6a-947e-11e5-858a-fafeb9770e35.png)

Show diff = on (raw diff)
![show diff - on](https://cloud.githubusercontent.com/assets/5120011/11429870/050f707c-947e-11e5-90e8-5139ef0b3c30.png)

**Comment by jakesankey** [comment]
Good idea. Thanks for the feedback. 

For more diff-goodness. Checkout line-diff-details package as well.

**Comment by irvnriir** [comment]
@jakesankey i'm pretty sure that on 2nd screenshot is not what supposed to be, and same issue here.

*n, 'added','removed' lines marked by "+","-"
